RELEASE.md

```diff
@@ -230,23 +239,23 @@ The Release Manager is also responsible for ensuring that each stakeholder tests
 
 The Technical Writer reviews the generated release notes by making adjustments directly on the corresponding Jira tickets and generating the release notes again to collect the most up-to-date information, including any cherry-picks that were done in the release branch.
 
-[Read more about the release notes process](https://codacy.slite.com/app/docs/X1IGdfa0vr1Rg4).
+[Read more about the release notes process](https://codacy.slite.com/app/docs/X1IGdfa0vr1Rg4) on Slite.
 
 ### Approval by the Release Manager
 
 If there are fixes for bugs identified during the testing stage, you must [patch the current release candidate](#patching-a-release) and restart the testing and stabilization stage with the new release candidate version.
 
 When all stakeholders have approved the release candidate, give a go-ahead of the release by clicking the Manual Approval step for QA and for the Solutions Engineers in the CircleCI workflow of your release candidate branch. After this Manual Approval on CircleCI the workflow will promote the release candidate to the [stable](https://charts.codacy.com/stable/api/charts) channel.
 
-## 4. Launching the new release
+## 5. Launching the new release
 
 The Release Manager must ensure that we have a stable release candidate and that both the QA team and relevant stakeholders have approved the release.
 
 Then, the Release Manager releases and announces the new version:
 
 -   [ ] 1.  If all is good give a public OK to the release
 
--   [ ] 2.  Tag the CLI and Coverage Reporter with the version of the release being done.
+-   [ ] 2.  Tag Codacy's [Analysis CLI tool](https://github.com/codacy/codacy-analysis-cli) and [CLI Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter) with the version of the release being done.
```

Comment:
Tweak to match the official name of the CLI

```suggestion
-   [ ] 2.  Tag the [Codacy Analysis CLI tool](https://github.com/codacy/codacy-analysis-cli) and [CLI Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter) with the version of the release being done.
```

Reply:
Yes, but even better would be to use the "official" names of both the Codacy Analysis CLI and Codacy Coverage Reporter. :nerd_face: 

```suggestion
-   [ ] 2.  Tag the [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli) and [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter) CLI tools with the version of the release being done.
```